frontend/benefit/applicant/.eslintrc.js

```diff
@@ -3,6 +3,8 @@ const { join } = require('path');
 module.exports = {
   extends: ['auto'],
   rules: {
+    'no-secrets/no-secrets': ['error', { tolerance: 4.2 }],
+    'no-unsanitized/method': 'off',
     'no-void': 'off',
```

Comment:
i would still enable no-void rule, you have still added the void to several places: https://github.com/City-of-Helsinki/yjdh/pull/73/files#diff-3e08260afd1fc2ba62e1377185fd239d85426f2b80622e2a103b375a4c8065dfR113

Reply:
i ll fix

Reply:
Oh I get mixed up. `'no-void' : 'off'` is good idea and use the `void` to resolve `@typescript-eslint/no-floating-promises` when floating promise is expected. So no need to do anything about it.